examples/worlds/buoyancy_engine.sdf

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" ?>
+<sdf version="1.6">
```

Comment:
Could you add some instructions here up top like we have on other example worlds? Copying the ones from the header should be enough